spec/overview.html

```diff
@@ -117,5 +117,36 @@ <h1>Tuple object</h1>
         </p>
       </emu-note>
     </emu-clause>
+
+    <emu-clause id="sec-box-value">
+      <h1>Box value</h1>
+      <p>
+        primitive value which contains an ECMAScript value
+      </p>
+      <emu-note>
+        <p>
+          A box is immutable and its will always contain the same value over time. However, the contents of such value can change if it is not immutable.

```

Comment:
this may just be my unfamiliarity, but the second sentence is confusing to me. i wasn’t sure what “it” in “if it is not immutable” was referring to (the contents of the value or the value itself). assuming it’s the contents of the value that is being referred to, this version makes that unambiguous:
```suggestion
          A box is immutable and it will always contain the same value over time. However, the contents of the value can change if the contents are not immutable.
```
another thing i was confused about is what the “contents of such value” refers to, but that seems more likely to be a matter of my unfamiliarity with what a Box is. i’m curious to know if there is a difference between “the contents of such value” and the “the value of the value”.

Reply:
An alternative wording could be
```suggestion
          A box is immutable and its will always contain the same reference over time. However, it does not guarantee that the referenced value will not be mutated.
```

i.e.
```js
const obj = { prop: 2 };
const box = Box(obj);
```

it will always be true that `box.unbox() === obj`, but `box.unbox().prop` could be different from `2` (if you do, for example, `obj.prop = 3`).

Reply:
i like that wording (“reference” and “mutable referenced values”). makes it much clearer to me. one nit: “and its will” should be “and it will”.